ocs_ci/utility/noobaa.py

```diff
@@ -0,0 +1,92 @@
+import base64
```

Comment:
As this is an OCS related resource, please move the file under ocs_ci/ocs/resources